components/ResultListPage/Header/index.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import User from "../../QuizCardPage/User";
+import User2 from "../../QuizCardPage/User2";
+
+export default function Header() {
+  return (
+    <StyledDiv>
+      <User1Container>
+        <User name="User1" aria-label="Smiley" emoji="😁" />
+      </User1Container>
+      <User2Container>
+        <User2 name="User2" aria-label="Smiley" emoji="😎" />
```

Comment:
Here you can use the `User` component instead to follow the DRY rule:
```suggestion
        <User name="User2" aria-label="Smiley" emoji="😎" />
```